discovery-infra/bootstrap_in_place.py

```diff
@@ -66,6 +66,7 @@ def installer_gather(ip, ssh_key, out_dir):
     os.remove(bundle_file_path) if os.path.exists(bundle_file_path) else None
 
 
+@utils.retry(exceptions=Exception, tries=3, delay=30)
```

Comment:
# We retry both with Python and with curl because sometimes `curl` doesn't retry specific errors